models/User.model.js

```diff
@@ -1,28 +1,35 @@
 const { Schema, model } = require("mongoose");
 
-// TODO: Please make sure you edit the User model to whatever makes sense in this case
 const userSchema = new Schema(
   {
-    username: {
-      type: String,
-      trim: true,
-      required: false,
-      unique: true
-    },
-    email: {
-      type: String,
-      required: true,
-      unique: true,
-      lowercase: true,
-      trim: true
-    },
-    password: {
-      type: String,
-      required: true
-    }
+    name: String,
+    age: Number,
+    totalActivities: Number,
+    totalTimeExercising: Number,
+    favoriteActivites: { type: String, enum: ['run', 'bike', 'swim', 'walk', 'other'] },
+    shoes: [String],
+    bikes: [String]
```

Comment:
here we will need posts property because a user can have multiple posts.
posts: [{ type: Schema.Types.ObjectId, ref: 'Post' }]